### ISSUE TITLE
Websocket handshake error:not a websocket handshake: ‘upgrade’ token not found in ‘Connection’ header

### ISSUE THREAD

**Issue by mingatcgi** [cloudfoundry/loggregator-release#351]
I have a fresh PCF SRT (1.12) installation on AWS. When I do a push of the sample Spring-Music sample application (https://github.com/cloudfoundry-samples/spring-music), I got this warning: 

"Timed out waiting for connection to Loggregator".  

And The log from /var/vcap/sys/log/loggregator_trafficcontroller on the control vm shows this error: 

2017/11/30 17:22:37 Connecting to 1 dopplers
2017/11/30 17:22:37 websocket handler: Not a websocket handshake: websocket: not a websocket handshake: ‘upgrade’ token not found in ‘Connection’ header
2017/11/30 17:22:37 Disconnecting from stream (10.0.4.6:8082) (doppler.disconnect=false) (ctx.disconnect=1)
2017/11/30 17:22:37 error receiving from doppler via gRPC context canceled

Any suggestion what would be the cause of the issue and how I can resolve it?  Thanks.

Thanks for your submission!

Here are some guidance for submitting your issue.

Tell us why you are submitting?

- [ ] I found a bug - here are some steps to recreate it.
- [ ] I have an idea for a new feature - please document as "As a user, I would
like to..."
- [x] I am having trouble getting setup, understanding documentation, or using
Loggregator in some way - please highlight any resources you are using.
- [ ] This is an architecture change that will result in cleaner more efficient
code - Tell us why you think this is a good idea.

We review issues and PR's on a weekly basis and try to schedule and prioritize
items. If you are wondering about status of an item feel free to contact us on
our [slack channel](https://cloudfoundry.slack.com/messages/C02HCCXV5).

**Comment by apoydence** [comment]
@mingatcgi It sounds like your PCF deployment is not configured correctly. I would reach out to PCF support.

**Comment by ahevenor** [comment]
@mingatcgi - are you still experiencing the issue? As Andrew mentioned if you are using PCF, feel free to escalate through Pivotal support. I am closing this issue.

**Comment by GeekyTex** [comment]
Found this issue via Google..

I ran into this exact exception for a similar action (failure to retrieve logs when doing a command like `cf logs <app_name>`)

Likewise, my environment was Pivotal CF on AWS. 

I know this is on Pivotal's side, but commenting in case anyone else stumbles across this issue as I did.

When deploying PCF to AWS, you *must* set/override the Loggregator port to 4443 ([per their documentation](https://docs.pivotal.io/pivotalcf/2-0/customizing/aws-er-config-terraform.html#networking))

Relevant doc:

> For Loggregator Port, you must enter 4443 In AWS deployments, port 4443 forwards SSL traffic 
> that supports WebSockets from the ELB. Do not use the default port of 443

Hope this helps anyone who skimmed the docs a little to quick the first time around like me!
